ruby-monday-blog/spec/models/post_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it "makes a new post with title" do
```

Comment:
I think the description should read "makes a new post with title and body" since it's testing for both. Also, I'm wondering if we should do a different test for the body. What do you think?